analyzer_test.go

```diff
@@ -28,8 +28,8 @@ import (
 )
 
 func TestAnalyzer(t *testing.T) {
+	spec.Run(t, "unit-new-analyzer/", testAnalyzerFactory, spec.Parallel(), spec.Report(report.Terminal{}))
```

Comment:
These tests directly set the platform api, so iterating over all supported apis is not needed.